pkg/components/components.go

```diff
@@ -286,6 +287,10 @@ func GetDeployment(version string, operatorVersion string, namespace string, rep
 									Name:  "MONITORING_SERVICE_ACCOUNT",
 									Value: "prometheus-k8s",
 								},
+								{
+									Name:  "RUNBOOK_URL_TEMPLATE",
+									Value: monitoring.GetRunbookURLTemplate(),
```

Comment:
Currently, in an upstream installation, this will get the upstream template (i.e. `https://kubevirt.io/monitoring/runbooks/`) , although `RUNBOOK_URL_TEMPLATE` is not set.
Is this misleading? better to have it empty in upstream?

Reply:
Now we have the template we are forced to put something

Reply:
but the env var won't necessarily be set (in U/S it's not).
So, in U/S, do we want this field to be the actual env var value (empty), or to have the value of the template being used (`https://kubevirt.io/monitoring/runbooks/`)?